packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -1641,6 +1708,42 @@ export class Vector3 {
         return this;
     }
 
+	/**
+	 * Gets the rotation on all axes from the current vector to a vector
+	 * @param vector the target vector
+	 * @param ref the reference vector to assign the result;
+	 * @returns the rotation in the form (pitch, yaw, roll)
+	 */
+	 public getRotationToVectorToRef(vector: DeepImmutable<Vector3>, ref: DeepImmutable<Vector3>): Vector3 {
+		return Vector3.RotationBetweenVectorsToRef(this, vector, Vector3.ZeroReadOnly, ref);
+	}
+
+	/**
+	 * Gets the rotation on all axes from the current vector to a vector
+	 * @param vector the target vector
+	 * @returns the rotation in the form (pitch, yaw, roll)
+	 */
+	public getRotationToVector(vector: DeepImmutable<Vector3>): Vector3 {
+		return Vector3.RotationBetweenVectors(this, vector, Vector3.ZeroReadOnly);
+	}
+
+	/**
+	 * Gets the spherical coordinates of the current vector
+	 * @param ref the reference Spherical to assign the result
+	 * @returns the spherical coordinates for the current vector
+	 */
+	public toSphericalToRef(ref: Spherical): Spherical {

```

Comment:
Same comment as earlier, let s keep those on the spherical side to prevent the extra import